_posts/2016-01-01-Markdown Reference.md

```diff
@@ -172,33 +174,35 @@ A colon on the left-most side indicates a left-aligned column; a colon on the ri
 
 ### Footnotes
 
-``` markdown
-You can create footnotes like this[^footnote].
+You can create footnotes like this[^fn1] and this[^fn2].
 
-[^footnote]: Here is the *text* of the **footnote**.
-```
+[^fn1]: Here is the *text* of the first **footnote**.
+[^fn2]: Here is the *text* of the second **footnote**.
 
 will produce:
 
-You can create footnotes like this[^footnote].
+``` markdown
+You can create footnotes like this[^fn1] and this[^fn2].
 
-[^footnote]: Here is the *text* of the **footnote**.
+[^fn1]: Here is the *text* of the first **footnote**.
+[^fn2]: Here is the *text* of the second **footnote**.
+```
 
-Hover over the ‘footnote’ superscript to see content of the footnote.
+Hover over the ‘fn’ superscript to see content of the footnote. You can use whatever unique identified you like as the footnote marker (e.g. "fn1").
```

Comment:
'fn' seems not used in this example, how about change from

> Hover over the ‘fn’ superscript to see content of the footnote.

to

"Hover over the ‘fn1’ and ‘fn2’ superscript to see content of the footnote." 

for better understanding?

Reply:
OK, have fixed.